cmd/tracee-ebpf/main.go

```diff
@@ -201,17 +216,15 @@ func main() {
 			if listenMetrics {
 				err := t.Stats().RegisterPrometheus()
 				if err != nil {
-					fmt.Fprintf(os.Stderr, "Error registering prometheus metrics: %v\n", err)
+					logger.Error("Registering prometheus metrics", logger.ErrorField(err))
```

Comment:
Maybe we should have a more structured string here instead like "prometheus_metrics" WDYT? Otherwise I think the previous "free" string would fit better.

Reply:
I would go with structured `logger.Error("prometheus_metrics", logger.ErrorField(err))`.